src/seedu/addressbook/data/AddressBook.java

```diff
@@ -65,6 +65,10 @@ public void clear() {
         allPersons.clear();
     }
 
+    public int size() {
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.